src/applications/caregivers/components/FormAlerts/SecondaryRequiredAlert.jsx

```diff
@@ -1,14 +1,26 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { focusElement } from 'platform/utilities/ui';
 
-const SecondaryRequiredAlert = () => (
-  <va-alert status="error" uswds>
-    <h3 slot="headline">We need you to add a Family Caregiver</h3>
-    <p>
-      We can’t process your application unless you add a Family Caregiver.
-      Please go back and add either a Primary or Secondary Family Caregiver to
-      your application.
-    </p>
-  </va-alert>
-);
+const SecondaryRequiredAlert = () => {
+  useEffect(() => {
+    focusElement('.caregiver-error-message');
+  }, []);
+
+  return (
+    <div className="caregiver-error-message">
+      <va-alert status="error">
+        <h3 slot="headline">You need to add or change a Family Caregiver</h3>
+        <p>
+          You didn’t add or change a Primary or Secondary Family Caregiver in
+          this application.
```

Comment:
🤔 We have several large chunks of text in-line here - what's the guidance on whether we put it in directly like this vs extracting it out to `content.json`?

Reply:
For me... `content.json` is for magic strings that would go in some logic or content that would be passed into a component (like a form field label or some hint text or something like). Since this is a content component (no different really than the description components), having the text in here makes sense until we get to a point where maybe we need to do translations or something along those lines.